scripts/cluster/agent.py

```diff
@@ -9,6 +9,7 @@
 import subprocess
 import sys
 import time
+from typing import List
```

Comment:
This is necessary due to `# type: List[str]` typing comment. It seems like we might want to just annotate `get_dqlite_voters()` instead.